spec/requests/staging_auth_spec.rb

```diff
@@ -11,7 +11,8 @@
 
   context 'when in staging' do
     before(:each) { stub_global_auth(return_value: true) }
-    it_behaves_like 'basic auth'
+    # it_behaves_like 'basic auth'
```

Comment:
See note on comment lint, above.  Actually, if we kill this completely, then we should probably just remove this test altogether.